lib/rubocop/cop/sorbet/binding_constants_without_type_alias.rb

```diff
@@ -17,7 +17,7 @@ module Sorbet
       #   FooOrBar = T.type_alias { T.any(Foo, Bar) }
       class BindingConstantWithoutTypeAlias < RuboCop::Cop::Cop
         def_node_matcher(:binding_unaliased_type?, <<-PATTERN)
-          (casgn _ _ [#not_nil? #not_t_let? #method_needing_aliasing_on_t?])
+          (casgn _ _ [#not_nil? #not_t_let? #not_type_member? #method_needing_aliasing_on_t?])
```

Comment:
Should we also check for `type_template`: https://sorbet.run/#%23%20typed%3A%20true%0A%0Aclass%20Box%0A%20%20extend%20T%3A%3AGeneric%0A%0A%20%20Elem%20%3D%20type_template%0Aend ?

Reply:
Probably - I can add that too. I didn't know that was a thing.

Reply:
https://github.com/Shopify/rubocop-sorbet/pull/38/commits/c33c7d92d27c5a992d877e6aead7bcf9c1d06613